docs/Configuration-Settings.md

```diff
@@ -406,6 +406,12 @@ number.
   <td>10000</td>
   <td>Timeout between worker and client connection indicating a lost session connection.  Specified in milliseconds</td>
 </tr>
+<tr>
+  <td>tachyon.worker.block.lock.count</td>
+  <td>1000</td>
+  <td>Total number of locks for a Tachyon block worker.</td>
```

Comment:
I think this description should be more explicit that these locks are block locks. Maybe something like: "The number of block locks..." Also, what are the implications for larger number or fewer number of block locks? Maybe that tradeoff should also be in this description. Thanks!